docs/features/shared-volumes-with-NFS.md

```diff
@@ -0,0 +1,129 @@
+# Supporting shared volumes with vSphere Docker Volumes
+
+## Requirements
+* Developer should able to use externally managed NFS share as “shared docker volume” using “vsphere” volume driver.
+* This allows them to use NFS directories as docker volumes using Docker APIs, Compose etc 
+* Extend the vSphere Docker Volume plugin to support NFS shared volumes and mutliple readers/writers access
+* Volumes are accessible on multiple docker hosts running on  multiple ESX hosts in a Docker swarm
+* Allow admin to specify what remote dirs are managed by the plugin
+* Support all current docker volume plugin API for shared volumes
+
+## User experience
+### Set up remote directories for docker volumes
```

Comment:
Yes the first three points are to set up the NFS server and export the necessary folders on it. (4) I've moved to a statement below its not a part of the list to set up the NFS server.